spec/serializers/web_hook_user_serializer_spec.rb

```diff
@@ -23,18 +23,13 @@
 
   it 'should only include the required keys' do
     count = serializer.as_json.keys.count
-    difference = count - 44
+    difference = count - 45
 
     expect(difference).to eq(0), lambda {
-      message = ""
-
-      if difference < 0
-        message << "#{difference * -1} key(s) have been removed from this serializer."
-      else
-        message << "#{difference} key(s) have been added to this serializer."
-      end
-
-      message << "\nPlease verify if those key(s) are required as part of the web hook's payload."
+      message = (difference < 0 ?
```

Comment:
When this error has failed in the past, the error is because the test itself broke; it did not say the error message it was supposed to. It was the cannot modify frozen string error.

Reply:
I believe this is unrelated and can be safely ignored.

Reply:
I added to the user serializer, so this test failed.

This is the error message I got:
```
Failures:
  1) WebHookUserSerializer should only include the required keys
     Failure/Error: message << "#{difference} key(s) have been added to this serializer."
     
     FrozenError:
       can't modify frozen String
     # ./spec/serializers/web_hook_user_serializer_spec.rb:34:in `block (3 levels) in <main>'
     # ./spec/serializers/web_hook_user_serializer_spec.rb:28:in `block (2 levels) in <main>'
```

With this change, the new error message would be (if I didn't fix the error)
```
Failures:

  1) WebHookUserSerializer should only include the required keys
     Failure/Error:
       expect(difference).to eq(0), lambda {
         message = (difference < 0 ?
                   "#{difference * -1} key(s) have been removed from this serializer." :
                   "#{difference} key(s) have been added to this serializer.") +
                   "\nPlease verify if those key(s) are required as part of the web hook's payload."
       }

       1 key(s) have been added to this serializer.
       Please verify if those key(s) are required as part of the web hook's payload.
     # ./spec/serializers/web_hook_user_serializer_spec.rb:28:in `block (2 levels) in <top (required)>'

```

Reply:
Oh right if you add to to the user serializer this is legit. I personally hate this test that fails every time we do that. You have to account for the new column in this test.